conformance/tests/backendtlspolicy.go

```diff
@@ -152,5 +157,100 @@ var BackendTLSPolicy = suite.ConformanceTest{
 					},
 				})
 		})
+
+		// Verify that changing a ConfigMap content should be reconciled by the controller
+		t.Run("Changing the content of a ConfigMap should be reconciled by the controller", func(t *testing.T) {
```

Comment:
nit: "Changing the content of a ConfigMap used by BackendTLSPolicy as CA certificate should be reconciled....